src/cache.ts

```diff
@@ -32,9 +32,24 @@ const supportedPackageManager: PackageManager[] = [
     path: [join(os.homedir(), '.gradle', 'caches'), join(os.homedir(), '.gradle', 'wrapper')],
     // https://github.com/actions/cache/blob/0638051e9af2c23d10bb70fa9beffcad6cff9ce3/examples.md#java---gradle
     pattern: ['**/*.gradle*', '**/gradle-wrapper.properties']
+  },
+  {
+    id: 'sbt',
+    path: [
+      join(os.homedir(), '.ivy2', 'cache'),
+      join(os.homedir(), '.sbt'),
+      getCoursierCachePath()
+    ],
+    pattern: ['**/*.sbt', '**/project/build.properties', '**/project/**.{scala,sbt}']
```

Comment:
According to https://www.scala-sbt.org/1.x/docs/GitHub-Actions-with-sbt.html#Caching, some files must not be cached (`~/.sbt/*.lock`, and `ivydata-*.properties`). Would it be possible to exclude them?

Reply:
I try to do this asap

Reply:
@julienrf To write the documentation (and my understanding), could you tell me why these files should be excluded?

Reply:
WIP: https://github.com/actions/setup-java/pull/332

Reply:
To be honest, I am not sure. @eed3si9n, could you please help us?

Reply:
I got some answers! See this thread: https://mobile.twitter.com/ericpeters/status/1367529292234588160 for an explanation of the `ivydata-*.properties` files. For the `*.lock` files, I guess these files are created by sbt to ensure that multiple instances of sbt don’t modify a shared resource at the same time, and these files should be deleted by sbt but may not be deleted in case sbt does not terminate correctly.

Reply:
@julienrf 

> I got some answers! See this thread: https://mobile.twitter.com/ericpeters/status/1367529292234588160 for an explanation of the `ivydata-*.properties` files. For the `*.lock` files, I guess these files are created by sbt to ensure that multiple instances of sbt don’t modify a shared resource at the same time, and these files should be deleted by sbt but may not be deleted in case sbt does not terminate correctly.

Perfect, thank you for these explanations ! 👍

What strategy should we use in your opinion?

- Let's document that we support caching from SBT 1.3.x only (coursier support) and simplify the code in that sense. 

- We support all versions with the moderate implementation complexity that goes with it.

Reply:
I would be conservative here and would recommend supporting all the versions. Also, it is still possible to turn Coursier off in recent versions of sbt by setting `useCoursier := false` in the build definition.

Reply:
Ok, let's do this

Reply:
Done https://github.com/actions/setup-java/pull/332